client/package.json

```diff
@@ -39,6 +41,7 @@
     "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-react-hooks": "^4.3.0",
     "jest": "^29.7.0",
-    "jest-environment-jsdom": "^29.7.0"
+    "jest-environment-jsdom": "^29.7.0",
+    "typescript": "5.2.2"
```

Comment:
Make sure these two dependencies are only in `devDependencies` and not `dependencies`, since they're only used in development. Otherwise they will also get installed when the production app gets built, making the built process slower.